project/src/views/new-evaluation/newevaluation.js

```diff
@@ -0,0 +1,412 @@
+const NewEvaluationPage = function() {
+    return `
+        ${NAV()}
+        <form action="/action_page.php" method="get">
+            ${CandidateDetailsForm()}
+            <div class="content">
+                ${TechnicalLevelPicker()}
+                ${Textarea()}
+                ${TechnicalAreaPicker()}
+            </div>
+        </form>
+        ${Footer()}
+    `;
+}
+
+const CandidateDetailsForm = function() {
+    return `
+    <div class="inputs inputs-skin">
+        <input class="inputs-candidate inputs-candidate-skin" type="text" name="Candidate" value="Candidate">
+        <input class="inputs-interviewer inputs-interviewer-skin" type="text" name="Interviewer" value="Interviewer">
+        <input class="inputs-date inputs-date-skin" type="date">
+    </div>
+    
+    `;
+}
+
+const TechnicalLevelPicker = function() {
+    return `
+        <div class="content-box">
+            <table class="content-box-tbl content-box-tbl-skin">
+            <caption class="content-box-tbl-tl content-box-tbl-tl-skin">Technical Level</caption>
+            <tr>
+                <th class="tbl-head tbl-head-skin">Trainee</th>
+                <th class="tbl-head tbl-head-skin">Junior</th>
+                <th class="tbl-head tbl-head-skin">Middle</th>
+                <th class="tbl-head tbl-head-skin">Senior</th>
+            </tr>
+            <tr class="second-half-of-table second-half-of-table-skin">
+                <td><input type="radio"></td>
+                <td><input type="radio"><input type="radio"><input type="radio"></td>
+                <td><input type="radio"><input type="radio"><input type="radio"></td>
+                <td><input type="radio"><input type="radio"><input type="radio"></td>
+            </tr>
+            </table>
+        </div>
+    `;
+}
+
+const Textarea = function() {
+    return `
+    <div class="content-box-same">
```

Comment:
call this comp TextareaSection and create a Textarea component that contains the html for a single textarea element and surrounding h2 and div and use it in here